v2/robotmk/src/bin/agent.rs

```diff
@@ -82,14 +83,14 @@ fn main() {
             return;
         }
     };
-    report_config_content(raw.clone());
     let config: Config = match serde_json::from_str(&raw) {
         Ok(config) => config,
         Err(e) => {
             report_config_error(e.to_string());
             return;
         }
     };
+    report_config_content(raw.clone());
```

Comment:
Having the config is particularly useful, if the de-serialization fails. Reporting the content if available was intentional on my end.

Reply:
As discussed: It's sufficient to report the config if the parsing suceeded.